### ISSUE TITLE
Duplicate channel

### ISSUE THREAD

**Issue by morlanth** [vexorian/dizquetv#9]
An option to duplicate channels would be usefull in creating channels which have the same content but are on a different shuffle cycles.

**Comment by vexorian** [comment]
Sounds like something that can be added in a later version

In the meantime, you can add your channel's contents to a plex playlist, then you load the playlist and apply the buttons, it's a bit faster than readding the whole channel.

**Comment by morlanth** [comment]
Yes, I’m already doing this using smart playlists for film genres.

The duplication is more for reducing the time by not having to reimport the contents of a playlist when creating multiple genre specific channels that contain the same content.

**Comment by MoRBiD-MaNGLeR** [comment]
I schedule duplicate channels of the same libraries on different shuffle cycles as well.  These libraries are already the equivalent of a filtered larger library.  I just have a lot of similar movies, and I dislike half a year going by before each movie has been played once.  The duplicate channels allows for greater playback frequency.

Until recently, I had the duplicate channels similarly branded in dizqueTV, but now I'm starting to give each one its own unique brand.  Very rarely are they playing the same content at the same time, let alone on the same day.

Anyway, please support this idea of simple channel duplication to save on channel-setup time over the current methods.
